examples/things_advanced.py

```diff
@@ -59,43 +58,42 @@ def process_request(self, req, resp):
             description = ('Please provide an auth token '
                            'as part of the request.')
 
-            raise falcon.HTTPUnauthorized(title='Auth token required',
-                                          description=description,
-                                          challenges=challenges,
+            raise falcon.HTTPUnauthorized('Auth token required',
+                                          description,
+                                          challenges,
                                           href='http://docs.example.com/auth')
 
         if not self._token_is_valid(token, account_id):
             description = ('The provided auth token is not valid. '
                            'Please request a new token and try again.')
 
-            raise falcon.HTTPUnauthorized(title='Authentication required',
```

Comment:
Using only keyword arguments is also by design, this was changed intentionally.

See also the background discussion & rationale here: https://github.com/falconry/falcon/pull/1655

Reply:
agreed